config/locales/es.yml

```diff
@@ -0,0 +1,11 @@
+es:
+  ibandit:
+    invalid_country_code: "'%{country_code}' no es un código de país ISO 3166-1 válido"
+    invalid_check_digits: "Los dígitos de control no han pasado la verificación de módulo. Se esperaba: '%{expected_check_digits}', recibido '%{check_digits}'."
+    invalid_length: "La longitud no casa con la specificación SWIFT (Se esperaba %{expected_length} símbulos, se ha recibido %{length})"
```

Comment:
Strange, in Spanish from Spain it doesn't sound right. Also definitions for casa as a verb is marriage. Might be more common in latinoamerican spanish